requirements.txt

```diff
@@ -1,18 +0,0 @@
-# Development, documentation & testing requirements.
-black
```

Comment:
Was it a deliberate decision to remove black?

Reply:
Yes, it's since been added to pre-commit and is defined and run via that.

One exception: `make lint-fix` runs Black and isort without pre-commit. We could add the `--help > /dev/null 2>&1 || ... pip install` thing? Although isort was never in `requirements.txt`

Reply:
See what you think of https://github.com/hugovk/Pillow/pull/80